src/planscape/forsys/scenario_sets.R

```diff
@@ -2,21 +2,24 @@ library(dplyr)
 library(forsys)
 library(sf)
 
-scenario_sets <- function(input_stand_data, priorities) {
-  input_stand_data
+scenario_sets <- function(input_stand_data, priorities, stand_id_field,
+                          proj_id_field, stand_area_field, stand_cost_field) {
+
+  scenario_output_fields <- append(priorities, c(stand_area_field, stand_cost_field))
```

Comment:
nit: better to use `c()` instead of `append()` since the latter wraps the former

Reply:
ah nice - done